### PR TITLE
local-pv-provisioner: v0.2.12

### PR DESCRIPTION
**Pull request by daichimukai**
Release local-pv-provisioner v0.2.12 that supports kubernetes v1.28.

### CODE REVIEW

File: local-pv-provisioner/local-pv-provisioner.yaml [hunk 1]
```diff
@@ -67,7 +67,7 @@ spec:
     spec:
       containers:
         - name: local-pv-provisioner
-          image: quay.io/cybozu/local-pv-provisioner:0.2.10
+          image: quay.io/cybozu/local-pv-provisioner:0.2.12
```
Comment (daichimukai) [inline local-pv-provisioner/local-pv-provisioner.yaml:70]: I forgot to update the version here when we released v0.2.11.

**Comment by llamerada-jp** [comment]
Versions written in Makefile should be updated.
https://github.com/cybozu/neco-containers/blob/935a9c33b6905dc530136b59f2dcbef4fa9d10b4/local-pv-provisioner/Makefile#L5-L6

**Review by llamerada-jp — CHANGES_REQUESTED** [review]
Versions written in Makefile should be updated.
https://github.com/cybozu/neco-containers/blob/935a9c33b6905dc530136b59f2dcbef4fa9d10b4/local-pv-provisioner/Makefile#L5-L6

**Comment by daichimukai** [comment]
> Versions written in Makefile should be updated.
> 
> https://github.com/cybozu/neco-containers/blob/935a9c33b6905dc530136b59f2dcbef4fa9d10b4/local-pv-provisioner/Makefile#L5-L6

Thank you. Fixed.
https://github.com/cybozu/neco-containers/compare/79bc1d4b5b937810b777983a7440e1d1d9120597..60d07a028295f979c2218e35110647e6feef52e9

**Review by llamerada-jp — APPROVED** [review]

**Pull request merged**